src/yang/ietf-bgp-policy.yang

```diff
@@ -292,11 +324,39 @@ module ietf-bgp-policy {
       description
         "Top-level container for BGP specific policy conditions.";
 
-      leaf med-eq {
-        type uint32;
+      container local-pref {
         description
-          "Condition to check if the received MED value is equal to
-           the specified value.";
+          "Value and comparison operations for conditions based on
+           the value of the BGP LOCAL_PREF.";
+        reference
+          "RFC 4271: Section 5.1.5.";
```

Comment:
Nit picking again, but you need the RFC title here.

Reply:
This keeps happening because I continue to copy and paste from elsewhere.  Please take up the normalization Issue as I work through the rest of the document changes.

Reply:
Mahesh: 
Do you need RFC 4271: A Border Gateway Protocol 4 (BGP-4), Section 5.1.5? 

Just curious.

Reply:
Resolved in next patch.

Reply:
Yes - I found the resolution in Issue #26.  He meant RFC 4271: A Border Gateway Protocol 4 (BGP-4), Section 5.1.5.